hatchery/paymodels.go

```diff
@@ -118,6 +119,20 @@ var getCurrentPayModel = func(userName string) (result *PayModel, err error) {
 		Config.Logger.Printf("Got error unmarshalling: %s", err)
 		return nil, err
 	}
+
+	if payModel.Local && Config.Config.Karpenter && strings.Contains(strings.ToLower(payModel.Name), "trial") {
```

Comment:
Is this correct? Only "Trial Workspaces" will have cost usage reports and any "Direct Pay" workspaces will not have cost usage reports?